.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,52 @@
+name: Bug report
```

Comment:
Perhaps delete https://github.com/devopness/devopness/blob/main/.github/ISSUE_TEMPLATE/bug_report.md as part of this same PR?

Reply:
Old issue templates deleted.